.github/workflows/tag_action.yml

```diff
@@ -3,25 +3,68 @@ name: Publish release
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+**'
 
 jobs:
   build:
     permissions: write-all
-    name: Add documentaiton as asset
+    name: Create release page
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v3
-      - name: Build
+
+      - name: Check artifacts from commit
+        id: get_artifacts
+        uses: dawidd6/action-download-artifact@v2
```

Comment:
[dawidd6](https://github.com/dawidd6) ?

Reply:
https://github.com/nrfconnect/sdk-nrf/blob/e7b53c3a17b61339fb144d5c9f8290bcd49c0ac0/.github/workflows/docpublish.yml#L15 I see nrf also uses that. so it's ok for me